NBitcoin/BitcoinCore/Coins.cs

```diff
@@ -88,58 +63,43 @@ public Coins(Transaction tx, int height)
 
 		private void UpdateValue()
 		{
-			_Value = vout.Where(o => !IsNull(o))
+			Value = Outputs.Where(o => !IsNull(o))
 							.Select(o => o.Value).Sum();
 		}
 
-		private bool IsNull(TxOut o)
-		{
-			return o.Value.Satoshi == -1;
-		}
-
-		public bool IsEmpty
-		{
-			get
-			{
-				return vout.Count == 0;
-			}
-		}
+		private bool IsNull(TxOut o) => o.Value.Satoshi == -1;

```

Comment:
**IsNull** is a private method and does not use ``this`` so, i think it could be declared as ``static``